packages/typescript/internal/ts_project.bzl

```diff
@@ -629,6 +637,7 @@ def ts_project_macro(
         declaration_map = compiler_options.setdefault("declarationMap", declaration_map)
         emit_declaration_only = compiler_options.setdefault("emitDeclarationOnly", emit_declaration_only)
         allow_js = compiler_options.setdefault("allowJs", allow_js)
+        resolve_json_module = compiler_options.setdefault("resolveJsonModule", resolve_json_module)
```

Comment:
you should also update the validator to check that the attribute setting matches what's in the tsconfig file

Reply:
thanks, fixed. are there any tests for the validator? i couldn't find any.

Reply:
not for the negative case, no